test/functional/page_objects/visualize_page.js

```diff
@@ -393,19 +393,25 @@ export function VisualizePageProvider({ getService, getPageObjects }) {
 
     // clickBucket(bucketType) 'X-Axis', 'Split Area', 'Split Chart'
     async clickBucket(bucketName) {
-      const chartTypes = await retry.try(
-        async () => await find.allByCssSelector('li.list-group-item.list-group-menu-item'));
-      log.debug('found bucket types ' + chartTypes.length);
-
-      async function getChartType(chart) {
-        const chartString = await chart.getVisibleText();
-        if (chartString === bucketName) {
-          await chart.click();
-          await PageObjects.common.sleep(500);
+      await retry.try(async () => {
+        const chartTypes = await retry.try(
+          async () => await find.allByCssSelector('li.list-group-item.list-group-menu-item'));
+        log.debug('found bucket types ' + chartTypes.length);
+
+        async function getChartType(chart) {
+          const chartString = await chart.getVisibleText();
+          if (chartString === bucketName) {
+            await chart.click();
+            await PageObjects.common.sleep(500);
```

Comment:
We cannot rely on a timer to be sure that the click is executed.

Reply:
what would be your suggestion ? anyway, i would keep that out of this PR ... in general i think we want to remove every sleep in our tests with something more reliable.